e2e/data.js

```diff
@@ -1,16 +1,16 @@
 const random = require('./helpers/random');
 const value = random(20);
 const data = {
-    server: 'https://ilarion.rocket.chat',
+    server: 'http://127.0.0.1:3000',
```

Comment:
This change makes tests to fail on CircleCI.
Can you run the docker on CI tests as well?